.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+    steps:
+      # 1. 소스 코드 가져오기
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      # 2. JDK 17 설정
+      - name: Set up JDK 17 with Gradle cache
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle' # gradle 의존성 캐싱, 이거 안 하면 한 번 받은 라이브러리는 캐시해두고 다음엔 바로 사용.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)